src/packages/hex.gleam

```diff
@@ -25,9 +84,23 @@ pub fn query() {
     json.decode(response.body, d.list(package_decoder()))
     |> result.map_error(JsonError)
 
-  io.debug(packages)
+  let new_packages =
+    packages
+    |> list.filter(fn(x) {
+      parse_iso8601_to_epoch_timestamp(x.updated_at) > last_ran
+    })
 
-  Ok(1)
+  case list.length(new_packages) >= list.length(packages) {
```

Comment:
Could we get some comments explaining the logic here?

Reply:
Added relevant comments